### PR TITLE
adds companies to motivation section

### PR DESCRIPTION
**Pull request by yati1998**
Fixes issue #861

### CODE REVIEW

**Comment by yati1998** [comment]
@alodahl I have made the changes. Please do review them.

**Comment by saberprashant** [comment]
Hi @yati1998 , you've only changed 5 files from the listed ones. Please strengthen the **Motivation Section** for other files too. Then, we'll review your PR.

**Comment by yati1998** [comment]
@saberprashant others file felt good to me. and hence didn't made any changes.

**Comment by yati1998** [comment]
@alodahl please do review my PR.

**Comment by alodahl** [comment]
> Hi @yati1998 , you've only changed 5 files from the listed ones. Please strengthen the **Motivation Section** for other files too. Then, we'll review your PR.

She checked with me in the issue comments - I agree these 4 don't need changes: financial literacy, resource-list, .env, negotiating

File: roles-in-tech/sales-engineering.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ None
 - The role of sales engineers involves translating and explaining highly complex technical information to customers and clients, focusing on revealing how a product or piece of equipment can solve specific problems. 
 -  They therefore play an important role in the modern sales process. 
 
+**Top companies hiring Sales engineers:**
```
Comment (alodahl) [inline roles-in-tech/sales-engineering.md]: 👏🏼

**Comment by yati1998** [comment]
> > Hi @yati1998 , you've only changed 5 files from the listed ones. Please strengthen the **Motivation Section** for other files too. Then, we'll review your PR.
> 
> She checked with me in the issue comments - I agree these 4 don't need changes: financial literacy, resource-list, .env, negotiating .

@alodahl do you want me to add examples in the file that are marked unchecked in the issue description?

File: professionalism/difficult-workplace-situations.md [hunk 2]
```diff
@@ -8,7 +8,17 @@ About 1 hour
 
 ### Motivation
 
-You will definitely have difficult conversations in your career! Learn here some tips for successfully navigating through those tough times.
+You will definitely have difficult conversations in your career! Hence it is important to learn here some tips for successfully navigating through those tough times.
+
+**Examples of some difficult situation one has to deal with in the workplace:**
+- You’re Overwhelmed With Work
```
Comment (alodahl) [inline professionalism/difficult-workplace-situations.md]: the capitalization is a bit distracting. Either:
1. only capitalize the first letter or each line
or
2. provide the same links for each line that are provided in the article.

File: mobile-development/mobile-development.md [hunk 3]
```diff
@@ -14,6 +14,18 @@ About 4 hours
 
 ### Motivation
 
+Mobile applications have revolutionized Mobile World, getting much more power in the hands of customer & more business in the pockets of the entrepreneurs. In the dynamic technology industry, mobile application development market is at its peak. Its is the need of the hour & the latest fruitful business trend.[(by Sara bhavsar)](https://www.quora.com/Why-is-the-mobile-application-development-important)
```
Comment (alodahl) [inline mobile-development/mobile-development.md]: Can you either:
1. put this section in quotes to show these are Sara's exact words
or
2. Fix the grammar - its a bit weird.

File: mobile-development/mobile-development.md [hunk 4]
```diff
@@ -14,6 +14,18 @@ About 4 hours
...
+
+Importance of mobile developement:
+- The World has gone mobile
+- Visible to Customers all the time
```
Comment (alodahl) [inline mobile-development/mobile-development.md]: 😎

File: mobile-development/mobile-development.md (hunk 3 above)
Comment (alodahl) [reply mobile-development/mobile-development.md]: Oh and capitalize her last name, please. Also on line 24.

File: databases/sql.md [hunk 5]
```diff
@@ -12,6 +12,10 @@
 
 Most software applications store data of some sort -- user data, content, etc. Databases are used to store data, so software developers should know how to use them. *Relational databases* are a common type of database that are good for storing many types of data.
 
+**Which companies use SQL database?**
+- [Microsoft](https://enlyft.com/tech/company/microsoft.com/)
+- [Nigelfrank](https://enlyft.com/tech/company/nigelfrank.com/)
```
Comment (alodahl) [inline databases/sql.md]: Nigel Frank

File: ajax/ajax.md [hunk 6]
```diff
@@ -16,6 +16,13 @@ About 3 hours
 ### Motivation
 Front-end code of interactive websites often need to talk to backend servers to get and present data that the user asks for. Usually this is done using API calls. AJAX is a way to make asynchronous calls to the server using JavaScript.
 
+**Examples of website using AJAX:**
+- [Digg.com(top of the page)](http://digg.com/)
+   - Instead of going to a login page, and then navigating back to the page you originally wanted, with AJAX, a user can type in their user name and password directly into the original page. From there AJAX will send a request to the server to log them in. The server let’s the page know they’ve been logged in, and the page you are on can update as needed. Digg has a login-system that works like this.
```
Comment (alodahl) [inline ajax/ajax.md]: Good work!  Please fix:
let's -> lets

File: ajax/ajax.md [hunk 7]
```diff
@@ -16,6 +16,13 @@ About 3 hours
...
+- [Google Search](https://www.google.com/)
+  - Google was one of the first major companies to start using AJAX, and Google’s search suggestion tool was one of the first ways they used it, and one of the first auto-complete tools made. When typing into the Google search bar, it starts to use AJAX to get common results from the database on each keystroke. Auto-Complete is great for forms where you have a lot of possible inputs, and making a select drop down would be too long and cumbersome.
```
Comment (alodahl) [inline ajax/ajax.md]: 💙

**Comment by yati1998** [comment]
@alodahl I have done the required changes. Please do review them.

File: databases/data-models.md [hunk 8]
```diff
@@ -7,7 +7,11 @@
 none
 
 ### Motivation
-To be able to plan out databases for projects.
+Every company needs database. Thus, it is important to be able to plan out databases for projects.
+
+**Top companies that rely on databases:**
+- [Microsoft](https://www.microsoft.com/)
+- [Gogole](https://www.google.com/)
```
Comment (alodahl) [inline databases/data-models.md]: Google

File: mobile-development/mobile-development.md [hunk 9]
```diff
@@ -13,7 +13,20 @@ About 4 hours
 - [React Lesson](/react-js/react.md)
 
 ### Motivation
+[(by Sara Bhavsar)](https://www.quora.com/Why-is-the-mobile-application-development-important)
```
Comment (alodahl) [inline mobile-development/mobile-development.md]: No, that is not what I meant.  Remove this line.  The paragraph needs formatting or grammar fixes.  Please see my previous comment.

File: professionalism/difficult-workplace-situations.md [hunk 10]
```diff
@@ -8,7 +8,17 @@ About 1 hour
...
-You will definitely have difficult conversations in your career! Learn here some tips for successfully navigating through those tough times.
+You will definitely have difficult conversations in your career! Hence it is important to learn here some tips for successfully navigating through those tough times.
+
+**Examples of some difficult situation one has to deal with in the workplace:**
+- You’re overwhelmed with work
+- Your boss quits or is fired
+- The job you have is different from the one you applied for
+- You keep running into conflict with a difficult co-worker
+- Your boss doesn’t notice the work you do
+- You’ve made a major mistake that truly harmed your team
```
Comment (alodahl) [inline professionalism/difficult-workplace-situations.md]: 🎯

**Comment by yati1998** [comment]
@alodahl I ahve made the changes. I hope it works this time.

**Pull request merged**